### PR TITLE
LINK-1314 |. Update API documentation

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Update documentation to much current API

## Closes
[LINK-1314](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1314)

[LINK-1314]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helevents/templates/rest_framework/event_list.html [hunk 1]
```diff
@@ -1,329 +1,411 @@
...
-<code>registration!</code>.
-<p>Example:</p>
-<pre><code>event/?registration!=true</code></pre>
-<p><a href="?registration!=true" title="json">See the result</a></p>
+        <h3 id="local-ongoing">Ongoing local events</h3>
+        <p>Use to quickly access local (municipality level) events that are upcoming or have not ended yet. 
+            Combines the search on a number of description, name, and keyword fields. Locality is defined on 
+            the basis of MUNIGEO_MUNI value, which is set in the settings file. In the Helsinki case all the 
+            events would be retrieved that happen within Helsinki. Comes in two flavors: AND and OR. 
+            Use <code>local_ongoing_AND=lapset,musiiki</code> to search for the events with both search
```
Comment (danipran) [inline helevents/templates/rest_framework/event_list.html:11]:
```suggestion
            Use <code>local_ongoing_AND=lapset,musiikki</code> to search for the events with both search 
```

**Review by danipran — APPROVED** [review]
I suppose it's fine? It's documentation, so can't really break. Not quite sure what was changed in the end since the indentation changes flag everything as changed. Have some random improvements I happened to spot while glancing through everything.

File: helevents/templates/rest_framework/event_list.html (hunk 1 above)
Comment (jorilindell) [reply helevents/templates/rest_framework/event_list.html]: Thanks for the suggestions and sorry about making tracking changes so difficult. Unindented html files were super difficult to read and some of them contained some extra closing tags etc. small issues. So decided to indent all the documents at the same time as updating missing query parameters to the documentation to make it more readable in the future.

Comment (danipran) [reply helevents/templates/rest_framework/event_list.html]: It's fine, finally someone fixes it 😁 I noticed the same a few months ago when I was adding some stuff

**Pull request merged**